ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1605,4 +1604,20 @@ FCU_DFSS_DFVSC_FDPM_CHWDC_SRC_DFRMM:
   - FDPM
   - CHWDC
   - SRC
-  - DFRMM 
\ No newline at end of file
+  - DFRMM
+
+FCU_RHDHS_DTM_DFSS_STM_DTC_SHM_SFVSC_ALM_SS_SRC:
+  description: "FCU with temperature control, humidity monitor with electrical parameter monitoring."
+  is_canonical: true
+  implements:
+  - FCU
+  - RHDHS
+  - DTM
+  - DFSS
+  - STM
+  - DTC
+  - SHM
+  - SFVSC
```

Comment:
You are mixing supply and discharge. Fix.

Reply:
noted sir. will do